com.avaloq.tools.ddk.checkcfg.ui/src/com/avaloq/tools/ddk/checkcfg/ui/outline/CheckCfgOutlineTreeProvider.java

```diff
@@ -52,7 +52,7 @@ protected Object _text(final ConfiguredCatalog configuredCatalog) {
     if (configuredCatalog.getCatalog() == null || configuredCatalog.getCatalog().eIsProxy()) {
       // Unresolved reference
       ruleText = new StyledString("<Unresolved catalog>", //
-      stylerFactory.createXtextStyleAdapterStyler(semanticHighlightingConfiguration.errorTextStyle()));
+          stylerFactory.createXtextStyleAdapterStyler(semanticHighlightingConfiguration.errorTextStyle()));

```

Comment:
Indentation on this file seems wrong now

Reply:
This indentation is due to the empty comment in the line above.